crud/insert.lua

```diff
@@ -8,7 +8,7 @@ local sharding = require('crud.common.sharding')
 local dev_checks = require('crud.common.dev_checks')
 local schema = require('crud.common.schema')
 
-local InsertError = errors.new_class('Insert',  {capture_stack = false})
+local InsertError = errors.new_class('InsertError',  {capture_stack = false})
```

Comment:
I'm okay generally, but are there clients, which lean on the old naming? @akudiyar, how about Java connector?

Reply:
We don't stick to any error names at the moment yet apart from the netbox' `ClientError`. But @ArtDu is doing some research now about the tarantool/crud errors in the scope of  https://github.com/tarantool/cartridge-java/pull/102.

@ArtDu please, take a look at changes in this PR.

Reply:
These changes have no effect on the code I was writing in tarantool/cartridge-java#102. Research with errors is still ongoing. So the changes are ok

Reply:
Thanks, Alexander, Alexey and Artem!